tests/bsim/bluetooth/samples/central_hr_peripheral_hr/boards/nrf5340bsim_nrf5340_cpuapp.conf

```diff
@@ -0,0 +1 @@
+CONFIG_BT_BUF_ACL_RX_SIZE=255
```

Comment:
Why do we need this both here and in the sample?

Reply:
updated with a comment in the file